### ISSUE TITLE
Failed to unmarshal Kubernetes API response after update to Akka Management 1.1.0

### ISSUE THREAD

**Issue by thjaeckle** [akka/akka-management#920]
### Versions used 

Akka version: 2.6.13
Akka Management version: 1.1.0

We are running Kubernetes 1.20 on managed Azure AKS.

### Expected Behavior

After updating from Akka Management 1.0.10 to 1.1.0 we would have expected to see no warnings in the logs.

### Actual Behavior

Actually, those WARNINGS are logged by `akka.discovery.kubernetes.KubernetesApiServiceDiscovery`:
```
Failed to unmarshal Kubernetes API response. 
Status code: [200 OK]; 
Response body: [HttpEntity.Strict(application/json,298268 bytes total)]. 
Ex: [Object is missing required member 'containerStatuses']

Resolve attempt failed! 
Cause: spray.json.DeserializationException: Object is missing required member 'containerStatuses'
```

The cluster formed itself, even with the WARNINGS.
Could it be that the `"containerStatuses"` is an optional field in the K8S API?

We don't use several containers per pod - as I understood the `"containerStatuses"` was added to support such a scenario.

**Comment by DennisVDB** [comment]
We had the same issue at work. In our case it was due to a pod not starting up (ready 0/2), when akka-management queries that pod there would no `containerStatuses` field, decoding would fail as expects at least an empty list. This could be solved by defaulting to an empty list of container statuses.

**Comment by thjaeckle** [comment]
Will the fix for this issue done in the merged PR #928 be released?

**Comment by octonato** [comment]
v1.1.2 is released and #928 is included: https://github.com/akka/akka-management/commit/bad23b9508ea67d9e553caf2a1a274da82446826
